Makefile

```
@@ -198,7 +198,7 @@ HELMIFY ?= $(LOCALBIN)/helmify
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v5.4.2
-CONTROLLER_TOOLS_VERSION ?= v0.15.0
+CONTROLLER_TOOLS_VERSION ?= 0.16.4
```

Need to add `v` to the beginning
```suggestion
CONTROLLER_TOOLS_VERSION ?= v0.16.4
```